tests/e2e/utils/flows.js

```diff
@@ -467,6 +467,7 @@ export const merchantWCP = {
 		await expect( page ).toClick(
 			'button.editor-post-publish-panel__toggle'
 		);
+		await page.waitFor( 500 );
```

Comment:
Since we've upgraded `@wordpress/e2e-test-utils` package to latest, we could replace `addNewPageCheckoutWCB` flow with the following code: (As shared on https://github.com/Automattic/woocommerce-payments/compare/e2e/update-blocks-test-flows)

```
        await page.on( 'dialog', async ( dialog ) => {
			await dialog.accept();
		} );

		await page.goto( WP_ADMIN_PAGES, {
			waitUntil: 'networkidle0',
		} );

		// Add a new page called "Checkout WCB"
		await expect( page ).toClick( '.page-title-action', {
			waitUntil: 'networkidle0',
		} );

		await page.waitForSelector( 'h1.editor-post-title__input' );
		await page.type( 'h1.editor-post-title__input', 'Checkout WCB' );
		await insertBlock( 'Checkout' );
		await publishPost();

		await page.removeAllListeners( 'dialog' );
```

This would help us remove code for pressing tab, escape, enter etc. :D

Reply:
Yep, I've implemented that before switching back to the original code, but `await publishPost()` being one line, didn't allow me to add a delay between the publish button click and the second confirmation click. We need it because there's an animation of opening the side drawer for reaching the second confirmation button, and without the delay, this causes puppeteer to hang after opening the side drawer. I'd like to keep it this way.